package.json

```diff
@@ -50,6 +51,8 @@
   ],
   "devDependencies": {
     "@intlify/eslint-plugin-vue-i18n": "~2.0.0",
+    "@poppinss/cliui": "^3.0.2",
```

Comment:
Please always use ~ for dev deps in my projects
That makes my projects more stable for developers